HLTrigger/Muon/plugins/HLTScoutingTrackProducer.cc

```diff
@@ -47,6 +47,7 @@ class HLTScoutingTrackProducer : public edm::global::EDProducer<> {
 
   const int mantissaPrecision;
   const double vtxMinDist;
+  const double ptMin;
```

Comment:
```suggestion
  const double ptMin_;
```
Please use underscore for class data members. I understand this convention was not followed prior to this PR for `vtxMinDist` and others, so if you could fix those too, it would be appreciated.

Reply:
Fixed [here](https://github.com/cms-sw/cmssw/pull/36840/commits/250c42e1cab0dbcd218000fa4fc2cd685426bbef).